api/v1alpha1/ndb_api_helpers.go

```diff
@@ -136,6 +136,7 @@ func GenerateProvisioningRequest(ctx context.Context, ndbclient *ndbclient.NDBCl
 	req.ActionArguments = append(req.ActionArguments, dbTypeActionArgs.GetActionArguments(dbSpec)...)
 
 	log.Info("Returning from ndb_api_helpers.GenerateProvisioningRequest", "database name", dbSpec.Instance.DatabaseInstanceName, "database type", dbSpec.Instance.Type)
+	log.Info("Database Provisioning", "Request:", req)
```

Comment:
Can we append this log (key-value) in the previous log statement?

Reply:
Tried it, this does not work because the interface for Log.Info & Log.Error are different

Reply:
But the statement on 138 is already working, right? 😳